packs/crown-of-the-kobold-king-bestiary/drazmorg-the-damned.json

```diff
@@ -1372,7 +1372,7 @@
                     "die": "d4"
                 },
                 "description": {
-                    "value": "<p>This long, gnarled staff looks like several bones fused together, then wrapped tightly with strips of desiccated, gray skin. At the top of the staff, a clump of sinew clasps a small stone orb resembling a large human eye. When the staff is activated, this eye twitches and glances around randomly. When wielding this staff, you gain a +1 item bonus to visual Perception checks.</p>\n<p><strong>Activate</strong> @UUID[Compendium.pf2e.actionspf2e.Item.Cast a Spell]</p>\n<p><strong>Effect</strong> You expend a number of charges from the staff to cast a spell from its list.</p>\n<ul>\n<li><strong>Cantrip</strong> @UUID[Compendium.pf2e.spells-srd.Item.Read Aura]</li>\n<li><strong>1st</strong> @UUID[Compendium.pf2e.spells-srd.Item.Sure Strike]</li>\n<li><strong>2nd</strong> @UUID[Compendium.pf2e.spells-srd.Item.Darkvision], @UUID[Compendium.pf2e.spells-srd.Item.Floating Flame] (creates what appears to be a large eye made of fire), @UUID[Compendium.pf2e.spells-srd.Item.See the Unseen]</li>\n<li><strong>3rd</strong> @UUID[Compendium.pf2e.spells-srd.Item.Organsight], @UUID[Compendium.pf2e.spells-srd.Item.Web of Eyes]</li>\n<li><strong>4th</strong> @UUID[Compendium.pf2e.spells-srd.Item.Clairvoyance], @UUID[Compendium.pf2e.spells-srd.Item.Countless Eyes], @UUID[Compendium.pf2e.spells-srd.Item.Detect Scrying]</li>\n</ul>\n<hr />\n<p><strong>Craft Requirements</strong> Supply one casting of all listed levels of all listed spells.</p>"
+                    "value": "<p>This long, gnarled staff looks like several bones fused together, then wrapped tightly with strips of desiccated, gray skin. At the top of the staff, a clump of sinew clasps a small stone orb resembling a large human eye. When the staff is activated, this eye twitches and glances around randomly. When wielding this staff, you gain a +1 item bonus to visual Perception checks.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply one casting of all listed levels of all listed spells.</p>"
```

Comment:
Might need flavor text for some spells - for example Drazmorg's Staff of All-Sight's Floating Flame "(creates what appears to be a large eye made of fire)".

Reply:
huh. Seems that's a feature we need. I'll start poking into it.

Reply:
We will probably need a detail field on the spells then.

Reply:
What are the spells that have this issue? Hell's staff seems to also need it. I'm wondering if we can omit it and put it in the description instead if there are few enough.

Reply:
Seems like we need it after all. I removed all the more complex staves from the update